### ISSUE TITLE
Cannot build

### ISSUE THREAD

**Issue by obartelt** [hajdam/jmzemu#1]
Sorry, if I'm maybe too stupid or inexperienced, but I've been trying and trying and trying and cannot get this to build. I have a JDK 8 and downloaded Gradle 6.8.1, but that lead to some errors, so I downloaded JDK 15, but still I get the following error on gradle build:

```
> Configure project :modules:jmzemu-device-api
Downloading https://github.com/exbin/exbin-framework-java/archive/modules/exbin-framework-api

FAILURE: Build failed with an exception.

* Where:
Script 'C:\Users\AMTANGEE\Downloads\jmzemu-master\modules\jmzemu-modules.gradle' line: 135

* What went wrong:
A problem occurred evaluating project ':modules:jmzemu-device-api'.
> Could not find method execute() for arguments [] on task ':modules:jmzemu-device-api:svnDownloadexbin-framework-api' of type at.bxm.gradleplugins.svntools.tasks.SvnExport.
```

It seems that task.execute() has been deprecated in Gradle 5, but then what should I use? I tried Gradle 5.6.4 before, which didn't work with JDK 15 at all. Also the github-URL with "/trunk/" always gives me a 404, when I try manually.

Any ideas? I just want to get a jar package out of this ;-)

**Comment by hajdam** [comment]
Hi, sorry to disappoint you, but the project is a complete mess and doesn't work at all...
There was just some work done on separation to modules and there is very basic test on stretching sample image to window, but that's all...
And I'm sorry that you wasted time with this... :-(

Building currently works only from local maven and requires manual dependency building with local publishing...
Current build with gradle 6.5.1 and JDK 11: http://sharpmz.zdechov.net/temp/jmzemu-0.2.0-SNAPSHOT.zip

Probably better approach would be to explain what are you looking for and maybe I can help you further...

**Comment by obartelt** [comment]
Yeah, I figured as much, as I started to port it to C# for a minute, and then saw what is all commented out or missing. Nevermind, I was just looking for a way to get a MZ-800 (my childhood computer :-)) working on my new M1 MacBook, but I managed to get a Windows version running on Wine. Thanks for the quick reply, though, and all the best!
